### PR TITLE
update README.md with loading prop

### PR DESCRIPTION
**Pull request by jruhland**
Sorry, forgot to update the README with information about the new prop, I realized it right after you merged.

### CODE REVIEW

**Comment by jruhland** [comment]
@hibiken

**Comment by hibiken** [comment]
@jruhland Would you mind submitting a PR for this by the end of the day?

File: README.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ class LocationSearchInput extends React.Component {
               })}
             />
             <div className="autocomplete-dropdown-container">
+              { loading && <div>Loading...</div>}
```
Comment (hibiken) [inline README.md]: please remove the whitespace before `loading`

Comment (hibiken) [reply README.md]: Actually nvm, I'll take care 👍

**Pull request merged**